lib/wings/active_fedora_converter/file_metadata_node.rb

```diff
@@ -9,7 +9,7 @@ def self.FileMetadataNode(resource_class) # rubocop:disable Naming/MethodName
         # skip reserved attributes, we assume we don't need to translate valkyrie internals
         schema = resource_class.schema.reject do |key|
           resource_class.reserved_attributes.include?(key.name) ||
-            key.name == :size
+            key.name == :size || key.name == :has_model
```

Comment:
This is needed to resolve this error when saving a resource repeatedly.

> You attempted to set the property `has_model' of  to a scalar value. However, this property is declared as being multivalued.

After the first save, ActiveFedora's attribute setter is used instead of ActiveTriple's which has filtering for has_model.

Reply:
gotcha, sounds good